package/MDAnalysis/topology/PDBQTParser.py

```diff
@@ -97,12 +95,13 @@ class PDBQTParser(TopologyReaderBase):
      - tempfactors
      - charges
 
-    Guesses the following:
-     - masses
 
     .. versionchanged:: 0.18.0
        Added parsing of Record types
-    .. versionchanged:: 2.7.0
+    .. versionchanged:: 2.8.0
```

Comment:
I think this is an accidental removal of the versionchanged for 2.7.0?

I.e. the Column 67-70 being ignored should be under 2.7.0 and then the rest under 2.8?

Reply:
sorry for this mistake!